packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -8,9 +8,38 @@
   </Alert>
 </template>
 
+<script lang="ts">
```

Comment:
Should we just put this in a separate file? I know this is valid but I find it strange to have multiple `<script>` tags in a single Vue file.

Reply:
I felt weird about doing it this way as well.  Looking at pulling this logic into the `useCohort` composable, although this logic is more implementation specific (even though it is the only implementation at the moment).

Reply:
Moved to useCohorts composable